Cargo.toml

```diff
@@ -0,0 +1,66 @@
+[workspace]
+members = ["bin/faucet"]
+
+resolver = "2"
+
+[workspace.package]
+authors      = ["Miden contributors"]
+edition      = "2024"
+exclude      = [".github/"]
+homepage     = "https://miden.xyz"
+license      = "MIT"
+readme       = "README.md"
+repository   = "https://github.com/0xMiden/miden-faucet"
+rust-version = "1.87"
+version      = "0.10.0"
+
+# Optimise the cryptography for faster tests involving account creation.
+[profile.test.package.miden-crypto]
+opt-level = 2
+
+[workspace.dependencies]
+# miden-node dependencies.
+miden-node-block-producer  = { branch = "next", git = "https://github.com/0xMiden/miden-node" }
+miden-node-proto           = { branch = "next", git = "https://github.com/0xMiden/miden-node" }
+miden-node-rpc             = { branch = "next", git = "https://github.com/0xMiden/miden-node" }
+miden-node-utils           = { branch = "next", git = "https://github.com/0xMiden/miden-node" }
+miden-remote-prover-client = { branch = "next", git = "https://github.com/0xMiden/miden-node" }
+
+# miden-base aka protocol dependencies. These should be updated in sync.
+miden-lib     = { branch = "next", git = "https://github.com/0xMiden/miden-base" }
+miden-objects = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-base" }
+miden-testing = { branch = "next", git = "https://github.com/0xMiden/miden-base" }
+miden-tx      = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-base" }
```

Comment:
Let's update these to v0.10 dependencies from crates.io for now.

Not for this PR: ideally, we'd replace these with a single `miden-client` dependency.